dashboard/src/app/stacks/stack-details/stack.controller.ts

```diff
@@ -101,7 +103,16 @@ export class StackController {
 
     $window.addEventListener('message', (event: {data: string}) => {
       if ('show-ide' === event.data) {
+        if (this.tmpWorkspaceId) {
+          //TODO remove this delay after fixing time of update tree
```

Comment:
Not yet. It is a time for project type update(from API). I should investigate it more.